0.0.1/agent.py

```diff
@@ -1,93 +1,188 @@
-
+import logging
```

Comment:
@RyanSept have you installed litellm locally? Which version?

Reply:
actually, would you mind to commit your changes to 'python3 -m pip freeze > requirements.txt'

Reply:
The only dependency I think I installed was nearai SDK. I have updated the requirements.txt file